packages/eas-cli/src/project/workflow.ts

```diff
@@ -34,3 +34,16 @@ export async function resolveWorkflowAsync(
   }
   return Workflow.MANAGED;
 }
+
+export type CombinedWorkflowData = {
+  [value in Platform]: Workflow;
+};
+
+export async function getCombinedWorkflowDataAsync(
+  projectDir: string
+): Promise<CombinedWorkflowData> {
+  return {
+    android: await resolveWorkflowAsync(projectDir, Platform.ANDROID),
+    ios: await resolveWorkflowAsync(projectDir, Platform.IOS),
+  };
```

Comment:
```suggestion
  const [android, ios] = await Promise.all([
    resolveWorkflowAsync(projectDir, Platform.ANDROID),
    resolveWorkflowAsync(projectDir, Platform.IOS),
  ]);
  return { android, ios };
```

Reply:
Why?

Reply:
Performance :D

Reply:
Can you alaborate?

Reply:
Ok, I'll ask another question. You changed it from `Promise.all` to the non-`Promise.all` version. Why?

Reply:
With `Promise.all` those two can run concurrently.

Reply:
Your implementation is running those methods one after another, Dominik's approach is running them at the same time.